src/main/java/com/wcc/platform/service/CmsService.java

```diff
@@ -70,6 +72,19 @@ public CollaboratorPage getCollaborator() {
     }
   }
 
+  /**
+   * Write Pojo Member to JSON.
+   */
+  public Member createMember(Member member) {
```

Comment:
createMember() - return void or  Member?

Reply:
I recommend you to return the object you created

Reply:
```suggestion
  public Member saveMember(Volunteer volunteer) {
```